api/src/test/resources/org/openmrs/include/standardTestDataset.xml

```diff
@@ -81,6 +81,7 @@
   <concept concept_id="792" retired="false" datatype_id="4" class_id="3" is_set="false" creator="1" date_created="2004-04-08 00:00:00.0" version="" changed_by="1" date_changed="2008-08-15 14:05:56.0" uuid="d144d24f-6913-4b63-9660-a9108c2bebef"/>
   <concept concept_id="5089" retired="false" datatype_id="1" class_id="1" is_set="false" creator="1" date_created="2004-08-12 00:00:00.0" version="" changed_by="1" date_changed="2005-02-25 11:43:43.0" uuid="c607c80f-1ea9-4da3-bb88-6276ce8868dd"/>
   <concept concept_id="5497" retired="false" datatype_id="1" class_id="1" is_set="false" creator="1" date_created="2004-08-12 00:00:00.0" version="0.1" changed_by="1" date_changed="2006-06-09 14:57:33.0" uuid="a09ab2c5-878e-4905-b25d-5784167d0216"/>
+  <concept concept_id="5584" retired="false" datatype_id="4" class_id="3" is_set="false" creator="1" date_created="2004-08-12 00:00:00.0" version="0.1" changed_by="1" date_changed="2006-08-12 00:00:00.0" uuid="6584d845-7584-11e4-945f-0858400c9s89"/>
```

Comment:
If you added this concept and the GO to OrderServiceTest-nonCodedDrugs.xml above instead of here, then you wouldn't need to touch existing tests in ConceptServiceTest and AdministrationServiceTest